Example_App/.gitignore

```diff
@@ -0,0 +1,25 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+.DS_Store
+
+# Generated by package manager
+node_modules/
+
+# Generated by Cordova
+
```

Comment:
I'm wondering if we should treat the `platforms` and `plugins` folders in the example app as 'generated' code. They should (theoretically) be re-added by `cordova prepare`, provided that the platforms and plugins are saved into the `config.xml` for the project.

It would greatly simplify the git structure and reduce churn for generated build artefacts etc.

What do you think?

Reply:
(perhaps if there's anything we want to customise like app icon this won't be so simple. Although IIRC from a past life as a Cordova application developer, we used some build step to cut & copy the assets into the correct folders at build time)